### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by seneca**
Small changes

### CODE REVIEW

File: packages/nuejs.org/docs/index.md [hunk 1]
```diff
@@ -34,18 +34,18 @@ Nue is a great fit for the following people:
...
 
 [image]
   small: /img/figma-to-css.png
   large: /img/figma-to-css-big.png
 
 
-With Nue, you can take advantage of a [global design system](global-design-system.html)  and [modern, minimalistic CSS](css-best-practices.html) to build websites with significantly less code than what you need with a JavaScript-based framework like **Next.js**. Your codebase can be so small, that it's in the ballpark of a [single Tailwind button](/blog/introducing-nue-css/). This changes the way you think about web development. Chances are that Nue turns you into a professional UX developer, who thinks more like a designer and less of an engineer.
+Nue allows you to take advantage of a [global design system](global-design-system.html)  and [modern, minimalistic CSS](css-best-practices.html) to build websites with significantly less code than with a JavaScript-based framework like **Next.js**. Your codebase can as small as a [single Tailwind button](/blog/introducing-nue-css/). This changes the way you think about web development. Chances are that Nue turns you into a professional UX developer, who thinks more like a designer and less than an engineer.
```
Comment (nobkd) [inline packages/nuejs.org/docs/index.md]:
```suggestion
Nue allows you to take advantage of a [global design system](global-design-system.html) and [modern, minimalistic CSS](css-best-practices.html) to build websites with significantly less code than with a JavaScript-based framework like **Next.js**. Your codebase can be as small as a [single Tailwind button](/blog/introducing-nue-css/). This changes the way you think about web development. Chances are that Nue turns you into a professional UX developer, who thinks more like a designer and less like an engineer.
```

File: packages/nuejs.org/docs/index.md [hunk 2]
```diff
@@ -34,18 +34,18 @@ Nue is a great fit for the following people:
...
 
 
 ### New levels of performance
-If there is one word to describe Nue, it's _minimalism_. There's order of magnitude less of everything: NPM modules, API methods, configuration options, and framework-specific idioms. This leads to significantly leaner websites with far less  HTML, CSS, and JavaScript on the final response. Just take any professionally-designed website and compare it with Nue, and you'll see the difference. For example:
+If there is one word to describe Nue, then it's _minimalism_. There's order of magnitude less of everything: NPM modules, API methods, configuration options and framework-specific idioms. This leads to significantly leaner websites with far less HTML, CSS and JavaScript generated from the final server response. Just take any professionally-designed website and compare it with Nue, and you'll see the difference. For example:
```
Comment (nobkd) [inline packages/nuejs.org/docs/index.md]:
```suggestion
If there is one word to describe Nue, then it's _minimalism_. There's order of magnitude less of everything: NPM modules, API methods, configuration options and framework-specific idioms. This leads to significantly leaner websites with far less HTML, CSS and JavaScript in the final server response. Just take any professionally-designed website and compare it with Nue, and you'll see the difference. For example:
```

File: packages/nuejs.org/docs/index.md [hunk 3]
```diff
@@ -107,7 +107,7 @@ Nue is a perfect system for freelancers and design agencies who want to offer th
 
 
 ### Try it
-You might not believe all the big claims until you [give Nue a try](installation.html). Chances are that you will start wondering why you have ever built websites in any other way.
+You might not believe all these big claims until you [give Nue a try](installation.html). Chances are you will start wondering why you have ever built websites in any other way.
```
Comment (nobkd) [inline packages/nuejs.org/docs/index.md]:
```suggestion
You might not believe all these big claims until you [give Nue a try](installation.html). Chances are, you will start wondering why you have ever built websites in any other way.
```

Just feels more like I would say it... Feel free to reject.

**Review by nobkd — CHANGES_REQUESTED** [review]

File: packages/nuejs.org/docs/index.md (hunk 1 above)
Comment (seneca) [reply packages/nuejs.org/docs/index.md]: I've just noticed that we need "be" just before "as small as" in this sentence.
Your change from "than" to "like" is great.

File: packages/nuejs.org/docs/index.md (hunk 2 above)
Comment (seneca) [reply packages/nuejs.org/docs/index.md]: 💯 agree.

File: packages/nuejs.org/docs/index.md (hunk 3 above)
Comment (seneca) [reply packages/nuejs.org/docs/index.md]: 👍

File: packages/nuejs.org/docs/index.md (hunk 1 above)
Comment (nobkd) [reply packages/nuejs.org/docs/index.md]: Good catch!

**Review by nobkd — APPROVED** [review]

File: packages/nuejs.org/docs/index.md [hunk 4]
```diff
@@ -19,13 +19,13 @@ Nue's content-first [development flow](ux-development.html) focuses solely on th
 - - -
 
 ## Target audience
-Nue is a great fit for the following people:
+Nue is a great fit for the following group of people:
 
 1. **UX developers**: who prefer to jump to CSS directly from **Figma** or **Sketch** without a [frustrating and time-consuming](//medium.com/design-warp/5-most-common-designer-developer-handoff-mishaps-ba96012be8a7) designer-developer handoff process in the way.
 
-2. **Beginner web developers**: who want to skip the redundant layers in the [frontend stack](//roadmap.sh/frontend) and start building websites quickly with modern HTML, CSS, and JavaScript.
+2. **Beginner web developers**: who want to skip the redundant layers in the [frontend stack](//roadmap.sh/frontend) and start building websites quickly with modern HTML, CSS and JavaScript.
```
Comment (tipiirai) [inline packages/nuejs.org/docs/index.md:26]: I'd like to merge this, but the oxford comma bothers me. Should be either everywhere, or not. 

https://www.scribbr.com/commas/oxford-comma/

Comment (seneca) [reply packages/nuejs.org/docs/index.md:26]: The Oxford comma is optional—it’s not a grammatical error to leave it out or to include it.
If you feel that it should be there, please include it.

**Pull request merged**